publication-doi-commons/src/test/java/no/unit/nva/publication/doi/dto/PublicationTest.java

```diff
@@ -48,6 +48,15 @@ void comparingTwoFullyPopulatedPublicationsInstancesIsEqual() {
         assertThat(publication, is(equalTo(identicalPublication)));
     }
 
+    @Test
```

Comment:
tests for when sameModifiedDate is true?

Reply:
updated